inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp

```diff
@@ -1982,446 +2022,117 @@ void MKLDNNInterpolateNode::createPrimitive() {
     if (getSelectedPrimitiveDescriptor() == nullptr)
         IE_THROW() << errorPrefix << " did not set preferable primitive descriptor";
 
-    auto jcp = jit_interpolate_config_params();
-    jcp.mode = mode;
-    jcp.src_dt = getParentEdgeAt(0)->getMemory().GetDataType();
-    jcp.dst_dt = getChildEdgeAt(0)->getMemory().GetDataType();
-    jcp.src_data_size = MKLDNNExtensionUtils::sizeOfDataType(jcp.src_dt);
-    jcp.dst_data_size = MKLDNNExtensionUtils::sizeOfDataType(jcp.dst_dt);
-    jcp.indices_size = sizeof(int);
-    size_t dimSize = dstDim.size();
-    auto srcDimPad5d = to5Dim(srcDimPad);
-    auto dstDim5d = to5Dim(dstDim);
-    jcp.OW = dstDim5d[4];
-    jcp.OH = dstDim5d[3];
-    jcp.OD = dstDim5d[2];
-    jcp.IW = srcDimPad5d[4];
-    jcp.IH = srcDimPad5d[3];
-    jcp.ID = srcDimPad5d[2];
-    jcp.spatial_dim_size = spatialDimSize;
+    const auto &srcDims = srcMemPtr->getStaticDims();
+    const auto &dstDims = dstMemPtr->getStaticDims();
+    const auto inPrc = getParentEdgeAt(0)->getMemory().getDesc().getPrecision();
+    const auto outPrc = getChildEdgeAt(0)->getMemory().getDesc().getPrecision();
+    setPostOps(attr, dstDims, true);
 
     if (getChildEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::ncsp)) {
-        jcp.layout = InterpolateLayoutType::planar;
+        configured_for_layout = InterpolateLayoutType::planar;
```

Comment:
In each inference call we have once layout. May be the initialization of `configured_for_layout` should be in `createPrimitive`? Because `hasLayoutType` has overheads we can avoid